src/metrics.ts

```diff
@@ -136,9 +136,23 @@ export async function getMoveDistanceSingleGame(game: FileReaderGame) {
   return {
     maxDistancePiece,
     maxDistance,
+    distanceMap
   };
 }
 
+export function getAverageDistance(distanceMap: { [key: string]: number }, gameCount: number ) {
```

Comment:
can you annotate this function? I'm actually not sure why it exists. Is this for all games or a single game?